static/bidder-info/evtech.yaml

```diff
@@ -1,19 +1,7 @@
 endpoint: "http://ads-pbs.direct.e-volution.ai/openrtb/{{.PublisherID}}?host={{.Host}}"
+aliasOf: "limelightDigital"
 maintainer:
   email: "engineering@project-limelight.com"
```

Comment:
I think this is same as parent and can be ommited.

Reply:
https://github.com/prebid/prebid-server/pull/3183/commits/e595fdde81f74391b512986fc1d7058ea0a0ef12

good catch